### PR TITLE
Add doc for builtins.functions

### PR DESCRIPTION
**Pull request by Quasimewdo**
List all implemented aggregate functions

### CODE REVIEW

File: greenplumpython/builtins/functions.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+            -----
+              45
+            -----
+            (1 row)
+
+    """
+    return FunctionExpr(aggregate_function(name="sum"), (arg,), group_by=group_by, db=db)
+
+
+def generate_series(
```
[the remark this answers is not in the document]
Comment (Quasimewdo) [reply greenplumpython/builtins/functions.py:149]: > This module contains a list of Functions methods that are currently available in GreenplumPython.

> The function in database that generates a series of values from start to stop

**Review by beeender — COMMENTED** [review]
Apply this patch to enable docstyle and example tests:

```
diff --git a/tox.ini b/tox.ini
index 5b5c1e0..34ea823 100644
--- a/tox.ini
+++ b/tox.ini
@@ -69,7 +69,8 @@ doctest_optionflags = NORMALIZE_WHITESPACE
 testpaths =
     tests
     greenplumpython/group.py
+    greenplumpython/builtins/functions.py

 [pydocstyle]
 # TODO: Enable docstyle check for all files
-match = .*(group|dataframe)\.py
+match = .*(group|dataframe|functions)\.py
```

**Review by beeender — APPROVED** [review]

File: greenplumpython/builtins/functions.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    return FunctionExpr(aggregate_function(name="avg"), (arg,), group_by=group_by, db=db)
+
+
+def sum(
+    arg: Any,
+    group_by: Optional[DataFrameGroupingSet] = None,
+    db: Optional[Database] = None,
+) -> FunctionExpr:
+    """
+    Compute the sum of a set of values.
```
Comment (yihong0618) [inline greenplumpython/builtins/functions.py:125]: Compute -> Calculate?

File: greenplumpython/builtins/functions.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    return FunctionExpr(aggregate_function(name="max"), (arg,), group_by=group_by, db=db)
+
+
+def avg(
+    arg: Any,
+    group_by: Optional[DataFrameGroupingSet] = None,
+    db: Optional[Database] = None,
+) -> FunctionExpr:
+    """
+    Compute the average value of a set.
```
Comment (yihong0618) [inline greenplumpython/builtins/functions.py:98]: Compute -> Calculate?

File: greenplumpython/builtins/functions.py [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+from greenplumpython.group import DataFrameGroupingSet
+
+
+def count(
+    arg: Optional[Any] = None,
+    group_by: Optional[DataFrameGroupingSet] = None,
+    db: Optional[Database] = None,
+) -> FunctionExpr:
+    """
+    Count the number of rows or non-NULL values against a specifield column or an entire table.
```
Comment (yihong0618) [inline greenplumpython/builtins/functions.py:15]: specifield -> specified

**Review by yihong0618 — APPROVED** [review]

**Pull request merged**